### PR TITLE
Merge to release branch for CD 14

### PR DESCRIPTION
**Pull request by luck3y**
Thanks for submitting your Pull Request!

Please make sure your PR meets the following requirements:

- [ ] Pull Request contains description of the issue
- [ ] Pull Request does not include fixes for other issues
- [ ] Attached commits represent units of work and are properly formatted
- [ ] You have read and agreed to the Developer Certificate of Origin (DCO) (see `CONTRIBUTING.md`)
- [ ] Every commit contains `Signed-off-by: Your Name <yourname@example.com>` - use `git commit -s`

### CODE REVIEW

File: cp-overrides.yaml [hunk 1]
```diff
@@ -0,0 +1,26 @@
+schema_version: 1
+
+# cp-overrides.yaml
+# see dev-overrides.yaml for more information
+
+labels:
+    - name: "PRODUCT_VERSION"
+      value: "13.0-SP01"
```
Comment (bstansberry) [inline cp-overrides.yaml:8]: 14?

File: cp-overrides.yaml [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+modules:
+      repositories:
+          - name: jboss-eap-modules
+            git:
+                  url: https://github.com/jboss-container-images/jboss-eap-modules.git
+                  ref: sprint-20
```
Comment (bstansberry) [inline cp-overrides.yaml:15]: Right sprint?

File: README.adoc [hunk 3]
```diff
@@ -1,4 +1,4 @@
-JBoss Enterprise Application Platform continuous delivery container image
+#JBoss Enterprise Application Platform continuous delivery container image
```
Comment (bstansberry) [inline README.adoc:1]: This is wrong, although I suppose it is in the eap-cd-dev branch. I know I fixed that somewhere, maybe in the openshift image?

Comment (luck3y) [reply README.adoc:1]: Ah, yeah vague memories of that, I'll find it. TBH, I'd just opened this, then got distracted by email, so I forgot about it, will fix it up a little and re-open I think.

Comment (bstansberry) [reply README.adoc:1]: https://github.com/jboss-container-images/jboss-eap-7-image/commit/672ef1c38b8912e43e75ffcf9adaf11ee5c5ecc5#diff-7bd4a925c695c2eb0eced3872b9b965f is what I was thinking of but that's the same as what you have here. I suspect it's wrong there, but anyway that all seems OT to this PR, which shouldn't be concerned with correcting issues in eap-cd-dev

**Pull request closed without merging**